.editorconfig

```diff
@@ -0,0 +1,22 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset and indentation style
+[*.{ts,tsx,js,html,css,scss,less}]
+charset = utf-8
+indent_style = space
+indent_size = 4
+
+# Matches the exact files, as these are using a different indentation style
+[{package.json,.tsconfig.json,tslint.json,.eslintrc.js}]
```

Comment:
This is changed in #953, so don't look too closely at it (or the typo with `.tsconfig.json`)